types/diffable-html/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for diffable-html 5.0
+// Project: https://github.com/rayrutjes/diffable-html#readme
+// Definitions by: Zhang Yi Jiang <https://github.com/DefinitelyTyped>
```

Comment:
this one mascarades your GitHub handle with DT handle? What's the reason behind this?

Reply:
Weird, I'm not sure how that happened - this was generated by the `dts-gen` command and I didn't notice it did this. I'll fix it